packages/frontend/src/components/wallet/ExploreNativeBanner.js

```diff
@@ -26,6 +29,22 @@ const StyledContainer = styled.div`
     }
 `;
 
+const StyledBannerLinkTemporary = styled.a`
```

Comment:
Can we skip creating a new styled component so that there's less cleanup later. You can use the `StyledBannerLink` for the external link by doing something like:

`<StyledBannerLink to={{ pathname: 'https://awesomenear.com' }} target='_blank' />`

Let's pass the URL directly without the variable so that it's super obvious what needs to be removed once the flag is removed

Reply:
Yes, definitely much cleaner this way